util/util.go

```diff
@@ -207,6 +208,36 @@ func RunTemplate(templateContent string, data interface{}, out io.Writer) error
 		"trimSuffix": func(content, suffix string) string {
 			return strings.TrimSuffix(content, suffix)
 		},
+		"cwd": func() (string, error) {
+			return os.Getwd()
+		},
+		"findLatestArtifact": func(content string) (string, error) {
```

Comment:
perhaps `findLatestFile`?  I think this routine should be a bit more generic since we dont actually know it will be used to find artifacts specifically, just files.